server/start.go

```diff
@@ -10,28 +10,28 @@ import (
 	"runtime/pprof"
 	"time"
 
-	"github.com/spf13/cobra"
-	abciclient "github.com/tendermint/tendermint/abci/client"
-	"github.com/tendermint/tendermint/abci/server"
-	tcmd "github.com/tendermint/tendermint/cmd/tendermint/commands"
-	tmos "github.com/tendermint/tendermint/libs/os"
-	tmservice "github.com/tendermint/tendermint/libs/service"
-	"github.com/tendermint/tendermint/node"
-	"github.com/tendermint/tendermint/rpc/client/local"
-	tmtypes "github.com/tendermint/tendermint/types"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
-
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/server/api"
 	"github.com/cosmos/cosmos-sdk/server/config"
-	servergrpc "github.com/cosmos/cosmos-sdk/server/grpc"
 	"github.com/cosmos/cosmos-sdk/server/rosetta"
-	crgserver "github.com/cosmos/cosmos-sdk/server/rosetta/lib/server"
 	"github.com/cosmos/cosmos-sdk/server/types"
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	servergrpc "github.com/cosmos/cosmos-sdk/server/grpc"
+	crgserver "github.com/cosmos/cosmos-sdk/server/rosetta/lib/server"
+	pruningTypes "github.com/cosmos/cosmos-sdk/pruning/types"
```

Comment:
I would go through all cases and make sure we don't camel-case this import

```suggestion
	pruningtypes "github.com/cosmos/cosmos-sdk/pruning/types"
```